### PR TITLE
Done with initial markup

### PR DESCRIPTION
**Pull request by mark-antony1**
Please try and play around with the app to see how react native works with very basic markup and shoot me any questions about it

### CODE REVIEW

File: client/Screens/HomeScreen.js [hunk 1]
```diff
@@ -15,23 +16,27 @@ export default class HomeScreen extends React.Component {
...
         <ScrollView style={styles.container}>
          <ListView
           dataSource={this.state.dataSource}
           //creates all the group activities dynamically 
           //with input from database
           renderRow={(rowData, i) => (
           /* Need to figure out how to map touchable elements */
-            <TouchableOpacity>
-              <Text key={i}>{rowData}</Text>
+            <TouchableOpacity style={styles.bordy}>
```
Comment (shermango) [inline client/Screens/HomeScreen.js]: is this suppose to be styles.body?

File: client/Screens/HomeScreen.js [hunk 2]
```diff
@@ -15,23 +16,27 @@ export default class HomeScreen extends React.Component {
     //generate rows that contain all current jam groups
     const ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});
```
Comment (shermango) [inline client/Screens/HomeScreen.js:17]: what does this comparator function do?

File: client/Screens/HomeScreen.js [hunk 3]
```diff
@@ -15,23 +16,27 @@ export default class HomeScreen extends React.Component {
...
+      dataSource: ds.cloneWithRows([1, 2, 3, 4, 5, 6, 6, 7, 9])
     };
   }
   render() {
     return (
       //essentially a div element
       <View style={styles.container}>
-        <Text>Your Groujkhvps</Text>
+        <Image 
+          source={{uri: 'https://cdn.shopify.com/s/files/1/0015/2602/files/jamzheaderrrr.jpg?v=1472243694'}}
```
Comment (shermango) [inline client/Screens/HomeScreen.js:65]: love the decal!

**Comment by mark-antony1** [comment]
i have a better pull request to merge

**Pull request closed without merging**